index.html

```diff
@@ -105,29 +105,26 @@
         made by me using the Python Programming Language. You can download it by
         clicking on the "Download" button given below. This app is free from any
         kind of virus and you can use it safely.
->>>>>>> 02388e92f3e05c12db30d6a503d2fa03877caf67
+
       </p>
     </center>
 
     <center>
-<<<<<<< HEAD
+
       <img class="img" src="assets/img/CALCULATOR.png" />
-=======
       <img class="img" src="assets/img/CALCULATOR.png" />
```

Comment:
Two images? @codesmith25103 can you close this pr and do the changes in another pr because some unwanted commits have been added to this pr ok?

Reply:
ok.. i will add another pr

Reply:
Shall I close this pr @codesmith25103 is it ok then